source/user-manual/capabilities/file-integrity/fim-fields-rule-mapping.rst

```diff
@@ -0,0 +1,145 @@
+
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  **Syscheck field**  |  **Alert field**            | **Field description**                                           |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  file                |  path                       |  File path in the current alert                                 |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  size                |  size_after                 |  File size in the current alert                                 |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  hard_links          |  hard_links                 |  List of hard links of the file                                 |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  mode                |  mode                       |  Syscheck scan mode                                             |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  perm                |  perm_after,win_perm_after  |  File permissions                                               |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  uid                 |  uid_after                  |  User ID of the process                                         |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  gid                 |  gid_after                  |  Group ID of the process                                        |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  md5                 |  md5_after                  |  md5 hash of the file in the current alert (after changes)      |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  sha1                |  sha1_after                 |  sha1 hash of the file in the current alert (after changes)     |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  uname               |  uname_after                |  User name of the process                                       |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  gname               |  gname_after                |  Group name of the process                                      |
```

Comment:
```suggestion
|  gname               |  gname_after                |  Group name of the group that shares ownership of the file                                      |
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/6f538ba0f29e9eb74ddfc9e363f4de470e21bb84)